RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -13,6 +15,12 @@
 
 
 
+_pfZeroSuppressionThresholds_EB_2018 = [0.140]*170
```

Comment:
these arrays show up with ```_B``` in the master
it would be better to match the name, especially if other "backports" happen to be needed in the future.